### PR TITLE
add a template tag for displaying lead_art

### PR DESCRIPTION
**Pull request by dmclain**

### CODE REVIEW

File: armstrong/apps/related_content/templatetags/related_content.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+@register.tag
+def lead_art(parser, token):
+    tokens = token.split_contents()
+    if len(tokens) is 3:
+        _, from_object, preset = tokens
+        return LeadArtNode(from_object, preset)
+
+    message = "Too %s parameters" % ("many" if len(tokens) > 3 else "few")
```
Comment (tswicegood) [inline armstrong/apps/related_content/templatetags/related_content.py:37]: This is technically correct, but could be made more useful by showing the usage.

File: armstrong/apps/related_content/templatetags/related_content.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class LeadArtNode(template.Node):
+    def __init__(self, from_object, preset):
+        self.from_object = template.Variable(from_object)
+        self.preset = template.Variable(preset)
+
+    def render(self, context):
+        obj = self.from_object.resolve(context)
+        obj_type = ContentType.objects.get_for_model(obj)
+        related_art = RelatedContent.objects.by_type('lead_art')\
+                                            .filter(source_type=obj_type)\
```
Comment (tswicegood) [inline armstrong/apps/related_content/templatetags/related_content.py:21]: Style note: traditionally this type of indention is done via an 8-space indention for the same reason pep8 frowns on dictionary value alignment.  Changing the model name ends up causes cascading changes that are just styling.

**Comment by tswicegood** [comment]
FWIW, I'm -0 on merging this.  I think we need a `{% render_related %}` that would do what this does and be more generally useful.  This can always been migrated to use the new code, but it's one more template tag to maintain when it only removes a few characters (example: `{% render_related some_object "lead_art" %}` versus `{% lead_art some_object %}`).

Given that this is going to undergo refactoring at some point very soon, it also needs tests that verify the anticipated behavior.

**Comment by tswicegood** [comment]
Just to round out this conversation.  We're going ahead and merging it with the intent of this being refactored heavily once a `{% render_related %}` and/or `some_model.related.lead_art` is implemented.  Right now it addresses a short-coming, so it's staying.  In the meantime, I might be persuaded that this is the correct way. :-)

**Comment by tswicegood** [comment]
This is ready to merge, pending tests for the template tags.

**Comment by dmclain** [comment]
dang it. that is very much not what i meant to do

**Pull request merged**